mesonbuild/interpreter/dependencyfallbacks.py

```diff
@@ -38,6 +42,30 @@ def __init__(self, interpreter: 'Interpreter', names: T.List[str], allow_fallbac
             if name in self.names:
                 raise InterpreterException('dependency_fallbacks name {name!r} is duplicated')
             self.names.append(name)
+        self.methods.update({'find_library': self.find_library_method,
+                             'has_function': self.has_function_method,
+                             'subproject': self.subproject_method,
+                             })
+
+    @permittedKwargs(find_library_permitted_kwargs - {'required', 'static'})
```

Comment:
If you are using `typed_pos_args` you might as well also use `typed_kwargs`.

Reply:
Because find_library() in Compiler object has not been ported yet, that should be done in another PR.